R/rd2roxygen.R

```diff
@@ -98,6 +102,9 @@ create_roxygen = function(info, usage = FALSE) {
     comment_tag("@author", info$author),
     comment_tag("@seealso", info$seealso),
     comment_tag("@references", info$references),
+    if(!is.null(info$concepts)) {
+      unname(sapply(info$concepts, function(x) comment_tag("@concept", x)))
+    },
```

Comment:
Perhaps just `unlist(lapply(info$concepts, function(x) comment_tag("@concept", x)))` so you don't need `if (is.null)`?